homeassistant/components/insteon/climate.py

```diff
@@ -46,7 +46,7 @@
     2: HVACMode.COOL,
     3: HVACMode.HEAT_COOL,
 }
-FAN_MODES = {4: HVAC_MODE_AUTO, 8: HVAC_MODE_FAN_ONLY}
+FAN_MODES = {4: FAN_AUTO, 8: FAN_ON}
```

Comment:
If we want to avoid a breaking change, we could introduce a new local constant for FAN_ONLY = "fan_only"